src/views/AfhandelingView.vue

```diff
@@ -417,6 +421,7 @@ import {
   type Contactmoment,
   koppelZaakContactmoment,
   CONTACTVERZOEK_GEMAAKT,
+ type saveContactmomentResponseModel,
```

Comment:
De conventie is om types te laten beginnen met een hoofdletter

Reply:
aangepast